exams/models_test.py

```diff
@@ -16,3 +19,15 @@ def test_exam_profile_str(self):  # pylint: disable=no-self-use
         ep = ExamProfile(id=1, status=ExamProfile.PROFILE_IN_PROGRESS)
 
         assert str(ep) == 'Exam Profile "1" with status "in-progress"'
+
+
+class ExamAuthorizationTest(TestCase):
+    """Tests for ExamAuthorizationss"""
+
+    def test_exam_authorization_str(self):  # pylint: disable=no-self-use
+        """
+        Test method ExamAuthorization.__str__ prints correctly
+        """
+        auth = ExamAuthorization(id=1, status=ExamProfile.PROFILE_IN_PROGRESS, user_id=2)
+
+        assert str(auth) == 'Exam Authorization "1" with status "in-progress" for user 2'
```

Comment:
nit: I personally think that we do not need to test the `__str__` method, but feel free to leave